apps/emqx_bridge_azure_event_hub/src/emqx_bridge_azure_event_hub.erl

```diff
@@ -162,7 +173,15 @@ ref(Name) ->
 
 producer_overrides() ->
     #{
-        authentication => mk(ref(auth_username_password), #{default => #{}, required => true}),
+        authentication =>
+            mk(
+                ref(auth_username_password),
+                #{
+                    default => #{},
+                    required => true,
+                    desc => ?DESC(auth_sasl_password)
```

Comment:
the DESC here should be `auth_sasl_password` or `auth_username_password`?

Reply:
It should actually be `"authentication"` here.